<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>411</width>
      <height>129</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="folerPathLine">
         <property name="placeholderText">
          <string>Path to folder</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="folderSelectButton">
         <property name="text">
          <string>Choose Folder</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="fileMaskEdit">
       <property name="placeholderText">
        <string>File mask</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="scanButton">
       <property name="text">
        <string>Scan Files</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="fileList">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
